.github/workflows/openstack.yml

```diff
@@ -150,10 +150,12 @@ jobs:
             username: 'root'
             key: ${{ secrets.SSH_PRIVATE_KEY }}
             port: '22'
+            timeout: 30m
+            command_timeout: 30m
             script: |
               cp -pv /opt/elevate/elevate-cpanel /scripts/elevate-cpanel
-              cp -pv /opt/elevate/.github/workflows/openstack/status_marker.sh /scripts/status_marker.sh
-              cp -pv /opt/elevate/.github/workflows/openstack/reboot_watch.sh /scripts/reboot_watch.sh
+              cp -pv /opt/elevate/.github/workflows/openstack/status_marker.pl /scripts/status_marker.pl
```

Comment:
Why end in pl or sh? That's not a relevant detail and just leads to more change like here I suggest stripping the extension

Reply:
Latest commit should have these removed.